frontend/src/scenes/toolbar-launch/AuthorizedUrlsTable.tsx

```diff
@@ -0,0 +1,178 @@
+    const columns: LemonTableColumns<KeyedAppUrl> = [
+        {
+            title: 'URLs',
+            dataIndex: 'url',
+            key: 'url',
+            render: function Render(url, record) {
+                const [urlUpdatingState, setUrlUpdatingState] = useState(record.url)
+                const [errorState, setErrorState] = useState('')
+                useEffect(() => setUrlUpdatingState(record.url), [record])
+                return record.type === 'suggestion' || (url !== NEW_URL && editUrlIndex !== record.originalIndex) ? (
+                    <div className={clsx('authorized-url-col', record.type)}>
+                        {record.type === 'authorized' && <CheckCircleFilled style={{ marginRight: 4 }} />}
+                        {url}
+                        {record.type === 'suggestion' && <LemonTag>Suggestion</LemonTag>}
+                    </div>
+                ) : (
+                    <div>
+                        <Input
+                            value={urlUpdatingState}
+                            onChange={(e) => setUrlUpdatingState(e.target.value)}
+                            onPressEnter={() => {
+                                setErrorState('')
+                                if (urlUpdatingState === NEW_URL) {
+                                    removeUrl(record.originalIndex)
+                                }
+                                // See https://regex101.com/r/UMBc9g/1 for tests
+                                if (
+                                    urlUpdatingState.indexOf('*') > -1 &&
+                                    !urlUpdatingState.match(/^(.*)\*[^\*]*\.[^\*]+\.[^\*]+$/)
+                                ) {
+                                    setErrorState(
+                                        'You can only wildcard subdomains. If you wildcard the domain or TLD, people might be able to gain access to your PostHog data.'
+                                    )
+                                    return
+                                }
+                                if (!isURL(urlUpdatingState)) {
+                                    setErrorState('Please type a valid URL or domain.')
+                                    return
+                                }
+
+                                if (
+                                    appUrls.indexOf(urlUpdatingState) > -1 &&
+                                    appUrls.indexOf(urlUpdatingState, record.originalIndex) !== record.originalIndex &&
+                                    appUrls.indexOf(urlUpdatingState, record.originalIndex + 1) !== record.originalIndex
+                                ) {
+                                    setErrorState('This URL is already registered.')
+                                    return
+                                }
+
+                                updateUrl(record.originalIndex, urlUpdatingState)
+                            }}
+                            autoFocus
```

Comment:
Autofocus has a tendency to open the keyboard on mobile, so I'm a bit worried about it being potentially annoying

Reply:
Done! We'll now only show it if you're not on mobile